features/networking/pod.feature

```diff
@@ -525,3 +525,79 @@ Feature: Pod related networking scenarios
       | ls /var/lib/cni/networks/openshift-sdn/ |
     Then the step should succeed
     And the output should not contain "10.132.2.200"
+
+  # @author zzhao@redhat.com
+  # @case_id OCP-41666
+  # @bug_id 1924741
+  @admin
+  @destructive
```

Comment:
I suggest also adding a `@slow` tag here for our future reference to filter test cases that consume a long time to execute.

Reply:
@jhou1 is there any special for this tag `@long-duration`

Reply:
No operations in ci, just wanted to enrich the categories of our tests.

Reply:
ok, got it. thanks